app/assets/stylesheets/components/c-projects-btn.scss

```diff
@@ -0,0 +1,23 @@
+.c-projects-btn {
```

Comment:
There are a lot of properties in here, I would see what common properties you can find between this and all your other buttons and then move them into something like .o-button then you can apply both classes to your buttons